exporter/collector/integrationtest/testdata/fixtures/ops_agent_self_metrics_expect.json

```diff
@@ -83,6 +108,29 @@
               }
             ]
           },
+          {
+            "metric": {
+              "type": "custom.googleapis.com/opencensus/grpc.io/client/completed_rpcs",
```

Comment:
I realize this is just the nature of how we did our self observabiltiy metrics, but it's highly confusing seeing the opencensus metrics here when Ops Agent defines its own self-observability metrics, that this test is measuring if we can write those self-observability metrics.

Reply:
Ya since this uses OpenCensus instrumentation we sort of have to use the OC exporter to capture them. Should we:
1. Tweak the metrics prefix config for the OC exporter or
2. Strip out the whole `custom.googleapis.com/opencensus/` prefix when saving the fixture